third-party/cmake/FetchRocksDB.cmake

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 4Paradigm
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set(SRC_URL https://github.com/facebook/rocksdb/archive/v6.27.3.tar.gz)
```

Comment:
@Leowner check whether `v6.27.3` is ok. If not, lower its version and update this `SRC_URL` and `URL_HASH`